nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
This is a comment more general than this PR but, in my opinion, using a versatile `cut_coords` is already misleading. I would add a `n_cuts` parameter to the plotting functions. This would solve the problem as we would be able to raise a meaningful error higher in the stack.